prime-angular-frontend/src/app/modules/site-registration/pages/business-licence/business-licence.component.html

```diff
@@ -28,6 +28,19 @@
           No business licence has been uploaded.
         </ng-container>
       </app-alert>
+
+      <ng-container *ngIf="businessLicenceDocuments[0]">
```

Comment:
If you're checking for uploaded documents you can using `businessLicenceDocuments.length` would be more explicit, otherwise you're suggesting you're specifically interested in the first index.